### DISCUSSION TITLE
Cannot save Theme customization changes

### DISCUSSION THREAD

**Discussion by dzanonuniv** [globaleaks/globaleaks-whistleblowing-software#3459] [Q&A]
### What version of GlobaLeaks are you using?

4.11.5

### What browser(s) are you seeing the problem on?

Firefox

### What operating system(s) are you seeing the problem on?

Windows

### Describe the issue

After selecting a favicon/CSS/files (what is "files" for anyway?) there is no "Save" button and I can't apply my changes.
If I select a favicon file and then go back to Settings and hit Save nothing changes.
Please advise.

Thanks

### Proposed solution

Implement a "Save" button or at least a message that says the favicon/CSS/file has been uploaded, also a preview for the favicon would be nice (as done with the Logo)

**Comment by evilaliv3** [comment]
Hello @dzanonuniv 

Actually if you load a file the file will be right away be loaded as configuration. Try to open the page in a new tab.

The only thing that you should consider is what already described in this discussion answer: https://github.com/orgs/globaleaks/discussions/3455

**Comment by dzanonuniv** [comment]
Hi @evilaliv3,
I already tried what you and the other discussion suggested but it doesn't work, the favicon simply isn't loaded.
I have only one site, trying loading a new page does nothing, loading the site via Tor doesn't show the favicon.
Can I load it manually somewhere? A config file perhaps?
The logo works correctly though, at least there's that.

Thanks

**Comment by erwindegrave** [comment]
@dzanonuniv I encountered that issue as well until I realised that the favicon needs to be .ico file, otherwise it does not accept it as a favicon.
Is that correct @evilaliv3 ?

**Comment by evilaliv3** [comment]
Correct @erwindegrave !
